topics/statistics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,104 @@
+
+This tutorial covers a simple example workflow illustrating how the multiGSEA package works. The omics data sets that will be used throughout the example were originally provided by Quiros et al. (Quirós et al. 2017). In their publication the authors analyzed the mitochondrial response to four different toxicants, including Actinonin, Diclofenac, FCCB, and Mito-Block (MB), within the transcriptome, proteome, and metabolome layer.
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Preparing the Data
+
+To perform pathway enrichment with MultiGSEA, you'll need omics datasets in the file type TSV . These datasets contain columns for feature Symbol, logFC pValue and adj.p-values. We'll use example data provided on Zenodo.
+
+## Get data
+
+> ### Data Upload
+>
+> 1. Create a new history for this tutorial.
+    {% snippet faqs/galaxy/histories_create_new.md %}
+> 2. Import the datasets from [Zenodo]({{ page.zenodo_link }})  into your Galaxy instance:
+>   - **transcriptomics.tsv**
+>   - **proteomics.tsv**
+>   - **metabolomics.tsv**
+>
+
+# Running MultiGSEA
+
+In this step, you'll use the MultiGSEA tool to perform GSEA-based pathway enrichment on the uploaded datasets.
+
+>## Selecting parameters
+><hands-on-title> Task description </hands-on-title>
+>
+> 1. Select the tool {% tool [multiGSEA](toolshed.g2.bx.psu.edu/repos/iuc/multigsea/multigsea/1.12.0+galaxy0) %} in Galaxy.
+> 2. Configure the input parameters as follows:
+>    - *"Select transcriptomics data"*: `Enabled`
+>        - {% icon param-file %} *"Transcriptomics data"*: `Transcriptomics`
+>    - *"Select proteomics data"*: `Enabled`
+>        - {% icon param-file %} *"Proteomics data"*: `Proteomics`
+>    - *"Select metabolomics data"*: `Enabled`
+>        - {% icon param-file %} *"Metabolomics data"*: `Metabolomics`
+> 3. You can also choose the Gene ID format for every data set. In this tutorial we will use the preset "SYMBOL" for transcriptomics and proteomics. For metabolomics we use HMDB.
+> 4. Select in **Supported organisms** the organism of which the data is about. In our case we select `Homo sapiens (Human)`.
+> 5. **Pathway databases**: Select relevant databases. For the tutorial we choose `KEGG`
+> 6. **Combine p-values method**: Choose a method (here `Stouffer` for balanced weighting).
+> 7. **P-value correction method** (for controlling false discovery rate): Choose `Holm`.
+> 8. Click on `Run Tool`
+>
+>    {% snippet faqs/galaxy/tools_run.md %}
```

Comment:
this snippet does not exist, could you check the name or remove (or if you wanted to add the faq perhaps you forgot to commit the file?)